model/github_app_auth.go

```diff
@@ -53,7 +53,7 @@ func HasGithubAppAuth(projectId string) (bool, error) {
 }
 
 // Upsert inserts or updates the app auth for the given project id in the database
-func (githubAppAuth *GithubAppAuth) Upsert() error {
+func (githubAppAuth GithubAppAuth) Upsert() error {
```

Comment:
I changed this to not be a pointer since it doesn't change any data when upserting- and it makes using it easier to chain

Reply:
I have a slight preference for keeping this a pointer because all of our Upsert functions (with the exception of one) are set up that way so I'd rather not break that precedent.